### PR TITLE
utils: fix ini and import tests from core

### PR DESCRIPTION
**Pull request by sergkudinov**
There was a issue in `packages/tools/src/repo.coffee.md` on line 113. It is using `ini.parse_multi_brackets`, but when I was moving to consistence `utils` I didn't notice that it is imported from `file.utils`.

### CODE REVIEW

**Comment by wdavidw** [comment]
If tools require files, then why involving the engine package ?

**Comment by sergkudinov** [comment]
@wdavidw It doesn't require @file. It requires `ini` utilities that were placed in `file` package and originaly in `core`. Since 2 packages (`file` and `tools`) are using same utilites, I decided to move it under `engine`.

**Comment by wdavidw** [comment]
Engine is not a trash bag ;)

**Comment by sergkudinov** [comment]
done

File: packages/tools/src/repo.coffee.md [hunk 1]
```diff
@@ -151,4 +151,4 @@ console.info(`Repo was updated: ${status}`)
 
 ## Dependencies
 
-    utils = require './utils'
+    utils = require "@nikitajs/file/lib/utils"
```
Comment (wdavidw) [inline packages/tools/src/repo.coffee.md:154]: `tools/package.json` doesn't declare `@nikitajs/file` as a dependency ?

**Review by wdavidw — CHANGES_REQUESTED** [review]

**Review by wdavidw — APPROVED** [review]

**Pull request merged**